docs/RecycleBin/RecycleBin.md

```diff
@@ -0,0 +1,19 @@
+## Recycle Bin
+
+Recycle Bin page lists all deleted accounts, cloudspaces, Virtual Machines (VMs), disks, images, and iso images unless deleted permanently. The Recycle Bin allows to recover objects within [`delete retention period`](#delete retention period), whereafter the objects are deleted permanently and cannot be restored.
+
+![](RecycleBin.png)
+
+For OVC objects that must have unique manes, like accounts, cloudspaces on the same account, and VMs on the same cloudspace, names stay reserved while objects stay in recycle bin. Therefore, same names cannot be used to create new accounts, cloudspaces and VMs, until the original objects are permanently deleted.
```

Comment:
Is this the current behaviour about the names? I thought we allow same names as stuff in the recycle bin but if you want to restore them you need to rename the existing ones?